learning/tour-of-beam/frontend/lib/components/difficulty.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+import '../constants/colors.dart';
+import '../constants/sizes.dart';
+
+enum DifficultyLevel { basic, medium, advanced }
+
+class Difficulty extends StatelessWidget {
```

Comment:
1. We call this "Complexity" in the docs.
2. Better to add "Widget" here to tell it from an `enum`.